nx-X11/programs/Xserver/hw/nxagent/Keyboard.c

```diff
@@ -1863,10 +1863,9 @@ void nxagentKeycodeConversionSetup(void)
     fprintf(stderr, "Failed to create the keyboard file\n");
   }
 
-  if (nxagentOption(ClientOs) == ClientOsLinux &&
-            drules != NULL && dmodel != NULL &&
-                (strcmp(drules, "evdev") == 0 ||
-                    strcmp(dmodel, "evdev") == 0))
+  if (drules != NULL && dmodel != NULL &&
```

Comment:
Yeah, it should be a compatible implementation. I missed the rules and model being the client-side values, so... without proper `evdev` support in the kernel, we're likely not going to see `evdev` anytime soon as these key's values any time soon. I guess it's fine then.